applications/FSIApplication/custom_utilities/ibqn_mvqn_convergence_accelerator.h

```diff
@@ -0,0 +1,455 @@
+ * @tparam TDenseSpace Linear algebra dense space
+ */
+template<class TSparseSpace, class TDenseSpace>
+class IBQNMVQNConvergenceAccelerator: public ConvergenceAccelerator<TSparseSpace, TDenseSpace>
+{
+public:
+
+    ///@name Type Definitions
+    ///@{
+    KRATOS_CLASS_POINTER_DEFINITION( IBQNMVQNConvergenceAccelerator );
+
+    typedef ConvergenceAccelerator<TSparseSpace, TDenseSpace>              BaseType;
+    typedef typename BaseType::Pointer                              BaseTypePointer;
+
+    typedef typename BaseType::DenseVectorType                           VectorType;
+    typedef typename BaseType::DenseVectorPointerType             VectorPointerType;
+
+    typedef typename BaseType::DenseMatrixType                           MatrixType;
+    typedef typename BaseType::DenseMatrixPointerType             MatrixPointerType;
+
+    typedef MVQNFullJacobianConvergenceAccelerator<TSparseSpace, TDenseSpace>          MVQNType;
+    typedef typename MVQNType::Pointer                                          MVQNPointerType;
+
+    ///@}
+    ///@name Life Cycle
+    ///@{
+
+    /**
+     * @brief Construct a new IBQNMVQNConvergenceAccelerator object
+     * MVQN convergence accelerator Json settings constructor
+     * @param rConvAcceleratorParameters Json string encapsulating the settings
+     */
+    explicit IBQNMVQNConvergenceAccelerator(
+        const double InitialOmega = 0.825,
+        const double AbsCutOff = 1e-8)
+        : mInitialOmega(InitialOmega)
+        , mpConvergenceAcceleratorLeft(Kratos::make_unique<MVQNType>(mInitialOmega, AbsCutOff, true))
+        , mpConvergenceAcceleratorRight(Kratos::make_unique<MVQNType>(mInitialOmega, AbsCutOff, true))
+    {
+    }
+
+    explicit IBQNMVQNConvergenceAccelerator(Parameters rParameters)
+    {
+        Parameters mvqn_default_parameters(R"({
+            "solver_type"            : "IBQN_MVQN",
+            "w_0"                    : 0.825,
+            "abs_cut_off_tol"        : 1e-8,
+            "interface_block_newton" : true
+        })");
+        rParameters.ValidateAndAssignDefaults(mvqn_default_parameters);
+
+        mInitialOmega = rParameters["w_0"].GetDouble();
+        const double abs_cut_off_tol = rParameters["abs_cut_off_tol"].GetDouble();
+        mpConvergenceAcceleratorLeft = Kratos::make_unique<MVQNType>(mInitialOmega, abs_cut_off_tol, true);
+        mpConvergenceAcceleratorRight = Kratos::make_unique<MVQNType>(mInitialOmega, abs_cut_off_tol, true);
+    }
+
+    /**
+     * Copy Constructor.
+     */
+    IBQNMVQNConvergenceAccelerator(const IBQNMVQNConvergenceAccelerator& rOther) = delete;
+
+    /**
+     * Destructor.
+     */
+    virtual ~IBQNMVQNConvergenceAccelerator(){}
+
+    ///@}
+    ///@name Operators
+    ///@{
+
+
+    ///@}
+    ///@name Operations
+    ///@{
+
+    void Initialize() override
+    {
+        KRATOS_TRY;
+
+        mpConvergenceAcceleratorLeft->Initialize();
+        mpConvergenceAcceleratorRight->Initialize();
```

Comment:
ahhhhhhhhhhhhhh
It uses two acc internally
didn't get that until now

Reply:
Yes. One takes care of the tractions while the other one does so for the displacements.